lib/render/user.php

```diff
@@ -24,14 +24,20 @@ function userAvatar(
     });
 
     if (!$user) {
+        $userSanitized = $username;
+        sanitize_outputs($userSanitized);
+
+        $iconLabel = '';
         if ($icon !== false && ($icon || !$label)) {
-            return '';
+            $iconLabel = "<img src='" . media_asset('/UserPic/_User.png') . "' alt='$userSanitized' width='$iconSize' height='$iconSize'>";
```

Comment:
Recommend adding a `title` tag, as the image has no meaning without an associated label or tooltip (code notes).

Reply:
Good call, added